applications/luci-app-hd-idle/po/ca/hd_idle.po

```diff
@@ -15,52 +15,74 @@ msgstr ""
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 "X-Generator: Pootle 2.0.6\n"
 
-#: applications/luci-app-hd-idle/luasrc/model/cbi/hd_idle.lua:15
+#: applications/luci-app-hd-idle/htdocs/luci-static/resources/view/hd_idle.js:22
+msgid "Add new hdd setting..."
+msgstr ""
+
+#: applications/luci-app-hd-idle/htdocs/luci-static/resources/view/hd_idle.js:27
 msgid "Disk"
 msgstr "Disc"
 
-#: applications/luci-app-hd-idle/luasrc/model/cbi/hd_idle.lua:13
+#: applications/luci-app-hd-idle/htdocs/luci-static/resources/view/hd_idle.js:24
 msgid "Enable"
 msgstr "Habilita"
 
 #: applications/luci-app-hd-idle/root/usr/share/rpcd/acl.d/luci-app-hd-idle.json:3
 msgid "Grant UCI access for luci-app-hd-idle"
 msgstr ""
 
-#: applications/luci-app-hd-idle/luasrc/model/cbi/hd_idle.lua:6
+#: applications/luci-app-hd-idle/htdocs/luci-static/resources/view/hd_idle.js:17
 #: applications/luci-app-hd-idle/root/usr/share/luci/menu.d/luci-app-hd-idle.json:3
 msgid "HDD Idle"
 msgstr "HDD Idle"
 
-#: applications/luci-app-hd-idle/luasrc/model/cbi/hd_idle.lua:7
+#: applications/luci-app-hd-idle/htdocs/luci-static/resources/view/hd_idle.js:17
 msgid ""
 "HDD Idle is a utility program for spinning-down external disks after a "
 "period of idle time."
 msgstr ""
 "HDD Idle és un programa per ralentitzar els discos externs després d'un "
 "període de temps inactiu."
 
-#: applications/luci-app-hd-idle/luasrc/model/cbi/hd_idle.lua:21
+#: applications/luci-app-hd-idle/htdocs/luci-static/resources/view/hd_idle.js:32
 msgid "Idle time"
 msgstr "Temps d'inactivitat"
 
-#: applications/luci-app-hd-idle/luasrc/model/cbi/hd_idle.lua:23
+#: applications/luci-app-hd-idle/htdocs/luci-static/resources/view/hd_idle.js:35
 msgid "Idle time unit"
 msgstr "Unitat de temps d'inactivitat"
 
-#: applications/luci-app-hd-idle/luasrc/model/cbi/hd_idle.lua:10
+#: applications/luci-app-hd-idle/htdocs/luci-static/resources/view/hd_idle.js:19
 msgid "Settings"
 msgstr "Ajusts"
 
-# Hours
-#: applications/luci-app-hd-idle/luasrc/model/cbi/hd_idle.lua:26
+#: applications/luci-app-hd-idle/htdocs/luci-static/resources/view/hd_idle.js:39
+msgctxt "Abbreviation for days"
+msgid "d"
+msgstr ""
+
+#: applications/luci-app-hd-idle/htdocs/luci-static/resources/view/hd_idle.js:38
+msgctxt "Abbreviation for hours"
 msgid "h"
-msgstr "h"
+msgstr ""
 
-# Minutes (not minimum)
-#: applications/luci-app-hd-idle/luasrc/model/cbi/hd_idle.lua:25
+#: applications/luci-app-hd-idle/htdocs/luci-static/resources/view/hd_idle.js:37
+msgctxt "Abbreviation for minutes"
 msgid "min"
-msgstr "min"
+msgstr ""
+
+#: applications/luci-app-hd-idle/htdocs/luci-static/resources/view/hd_idle.js:36
+msgctxt "Abbreviation for seconds"
+msgid "s"
+msgstr ""
+
+# Hours
+#~ msgid "h"
+#~ msgstr "h"
+
+# Minutes (not minimum)
+#~ msgid "min"
+#~ msgstr "min"
```

Comment:
It seems like that the `i18n-update.pl` can't handle this properly after adding the "msgctxt".

Reply:
Thats okay, the translations and source strings were the same anyway, so nothing lost.

Reply:
OK, thanks.